### DISCUSSION TITLE
how to enable createRoot

### DISCUSSION THREAD

**Discussion by zhihengGet** [vercel/next.js#34873] [Help]
i have reactRoot:true in next config 

<prev>
module.exports = {
  reactStrictMode: true,

  experimental: {
    reactRoot: true,
  },
};

</prev>

but  receving this...

        
       Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more

**Comment by lucksp** [comment]
did you find a way to do this?  Are you using Next13?
